python-avd/pyavd/_eos_designs/structured_config/network_services/vlan_interfaces.py

```diff
@@ -126,7 +126,7 @@ def _check_virtual_router_mac_address(vlan_interface_config: dict, variables: li
                 vlan_interface_config["pim"] = {"ipv4": pim_config_ipv4}
 
         # Only set VARPv6 if ipv6_address is set
-        if vlan_interface_config["ipv6_address"] is not None:
+        if vlan_interface_config["ipv6_address"] is not None or vlan_interface_config["ipv6_enable"]:
```

Comment:
adjusted the comment. Yes it implies that condition in 134 does match more often, but in my opinion that is the correct way as if in CLI you configure "ipv6 enable" and "ipv6 virtual router address..." it will activate VARPv6. Additionally I think in line 134 we have a mistake today as it checks the IPv4 VARP but should check the IPv6 VARP. I fixed that.